src/sentry/api/endpoints/group_details.py

```diff
@@ -248,7 +250,7 @@ def get(self, request, group):
                 'pluginActions': action_list,
                 'pluginIssues': self._get_available_issue_plugins(request, group),
                 'pluginContexts': self._get_context_plugins(request, group),
-                'userReportCount': UserReport.objects.filter(group=group).count(),
+                'userReportCount': UserReport.objects.filter(group=group, environment_id=environment_id).count(),
```

Comment:
Hey there @lynnagara ! :) I can see what you're doing here, and it's not quite right. When the environment is invalid, you'll end up looking for all UserReports that don't have an environment id. Instead, what we want to do, when the environment is invalid, is to return an empty queryset (or in this case a count of 0).